src/components/Sum.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
Create a separate folder for this component. For example `src/components/Sum/Sum.jsx`